pkilint/rest/__init__.py

```diff
@@ -114,3 +114,17 @@ def ocsp_response_lint(doc: model.OcspResponseModel) -> model.LintResultList:
     parsed_doc = doc.parsed_document
 
     return _OCSP_PKIX_LINTER.lint(parsed_doc)
+
+@app.get('/crl/pkix')
```

Comment:
Can we rename this endpoint to `/crl/pkix/crl`? The second "crl" is the category/type of PKIX CRL. This will allow us to add linters for PKIX ARLs, scoped CRLs, etc. in the future without changing the endpoint path structures.